src/convertToObject.js

```diff
@@ -7,6 +7,24 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const obj = {};
+
+  const strings = sourceString.split(';').filter(Boolean);
+
+  strings.forEach((string) => {
+    const [styleName, ...definitions] = string.split(':');
+
+    if (styleName && definitions.length > 0) {
+      const key = styleName.trim();
+      const value = definitions.join(':').trim();
+
+      if (key && value) {
+        obj[key] = value;
+      }
+    }
+  });
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. Consider using `map` or `reduce` to transform the array of strings into the final object.